CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 # Change Log
 All notable changes to COMMIT will be documented in this file.
 
-## [1.4.6] - 2021-03-25
+## [1.4.6] - 2021-04-23
```

Comment:
Shall we merge this PR inside v1.5.0?

Reply:
Maybe, depending on which PR is merged first.